fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -955,8 +955,8 @@ ACTOR Future<Void> tLogCommit(Reference<TLogGroupData> self,
 			qe.storageTeams.push_back(message.first);
 			qe.messages.push_back(message.second);
 		}
-		// Currently only store commit messages in memory and not using persistent queue
-		// self->persistentQueue->push(qe, logData);
+		self->persistentQueue->push(qe, logData);
+		self->persistentQueue->commit();
```

Comment:
I have to manually commit here, since there is no commit happening without it.

it seems we have logic below

```
				    std::max(logData->queueCommittingVersion, logData->queueCommittedVersion.get()) + 1))) {
					while (self->queueCommitBegin != self->queueCommitEnd.get() &&
					       !self->largeDiskQueueCommitBytes.get()) {
						wait(self->queueCommitEnd.whenAtLeast(self->queueCommitBegin) ||
						     self->largeDiskQueueCommitBytes.onChange());
					}
```

do we have a easy way for the test code to notify the tlog to commit?